code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -1,4 +1,5 @@
 #define EMPOWERED_THRALL_LIMIT 5
+var/in_jaunt = 0
```

Comment:
Not a fan of declaring "in_jaunt" in the global scope. Is this safe here @Carbonhell @TDcoolguy300 @Crystalwarrior  ?